lib/tether/tether.go

```diff
@@ -251,20 +252,32 @@ func (t *tether) setNetworks() error {
 }
 
 func (t *tether) setMounts() error {
+	// provides a lookup from path to volume reference.
+	NameLookupMap := make(map[string]string, 0)
```

Comment:
Changed this to `pathIndex` I kept the targetRef for logging. I think it is useful to us to have that information in the logs. It is better to have this mapping for the lookup rather than attempt to work backwards to find the name again in this instance.